modules/core/src/main/scala/com.snowplowanalytics.snowplow.lakes/processing/Processing.scala

```diff
@@ -61,13 +61,19 @@ object Processing {
     originalBytes: Long
   )
 
+  private case class Transformed(
+    events: List[Row],
+    schema: StructType
+  )
+
   private def eventProcessor[F[_]: Async: RegistryLookup](
     env: Environment[F]
   ): EventProcessor[F] = { in =>
     val resources = for {
       now <- Stream.eval(Sync[F].realTime)
```

Comment:
I think in `WindowState.build` we also get current time to set window start time? Perhaps we could get rid of this one and make this start timestamp field in `WindowState` case class?